lib/utapi/utapiReplay.js

```diff
@@ -0,0 +1,12 @@
+const UtapiReplay = require('utapi').UtapiReplay;
+const _config = require('../Config').default;
+
+// start utapi server
+export default function main() {
+    let config;
```

Comment:
We no longer need this to be a conditional since config for utapi is either defined or not. If it's not defined and replay is started, Utapi will throw an assertion.